Automattic/wp-calypso
# Legacy FSE editor integration removal

The frontend will still render, but no more template editing in the Editor.

Just a beautiful sea of red.

#### Changes proposed in this Pull Request

* No more Legacy Dotcom FSE editing!

#### Testing instructions
Try it out and see the editor disappear while the frontend renders normally

Related to #56975
